src/components/LetterLoad.vue

```diff
@@ -83,6 +92,9 @@ export default {
 
       return ''
     }
+  },
```

Comment:
I think we could use a watcher here for great effect. I started to port this code over and did something like this:

```
watch: {
    customMessage(newMessage) {
      console.log(newMessage)
      this.$store.commit('setGenericValue', { key: 'userCustomMessage', value: newMessage })
    }
  }
```

The watcher will  then update the Vuex store everytime the customMessage variable changes.

Reply:
I also propose renaming the Vuex key to be slightly more explicit.

Reply:
I have added the watcher and renamed the key as suggested. Hope it is fine.